package.json

```diff
@@ -14,6 +14,7 @@
     "react-router-dom": "^6.7.0",
```

Comment:
## 1
![image](https://user-images.githubusercontent.com/93039359/214757657-55c06bc1-65a1-4c0c-b4b4-dadd85897e27.png)
.
## 2.
![image](https://user-images.githubusercontent.com/93039359/214757679-e876179d-49cb-4143-ba2f-8030055e31f2.png)

## 3.
![image](https://user-images.githubusercontent.com/93039359/214757723-dc5af1a1-1b11-475a-9140-33cb08330d4a.png)

- [ ] Kindly remove these instructions from your readme file to make your readme file professional and look clean :+1: